config/manager/manager.yaml

```diff
@@ -39,4 +58,15 @@ spec:
         image: quay.io/mongodb/mongodb-kubernetes-operator:0.6.2
         imagePullPolicy: Always
         name: mongodb-kubernetes-operator
+        resources:
+          limits:
+            cpu: 1100m
+            memory: 1Gi
+          requests:
+            cpu: 500m
+            memory: 200Mi
```

Comment:
[q] How did we derive these values?

Reply:
According to Cian these are the values used for the enterprise operator

Reply:
makes sense, I guess at some point we'd want to perform some actual testing of the community operator to come with a precise number, but definitely out of scope here